linera-views/src/backends/scylla_db.rs

```diff
@@ -715,6 +718,30 @@ impl AdminKeyValueStore for ScyllaDbStoreInternal {
         Ok(namespaces)
     }
 
+    async fn list_root_keys(
+        config: &Self::Config,
+        namespace: &str,
+    ) -> Result<Vec<Vec<u8>>, ScyllaDbStoreInternalError> {
+        Self::check_namespace(namespace)?;
+        let session = SessionBuilder::new()
+            .known_node(config.uri.as_str())
+            .build()
+            .boxed()
+            .await?;
+        let query = format!("SELECT root_key FROM kv.{} ALLOW FILTERING", namespace);
+
+        // Execute the query
+        let rows = session.query_iter(query, &[]).await?;
+        let mut rows = rows.rows_stream::<(Vec<u8>,)>()?;
+        let mut root_keys = HashSet::new();
```

Comment:
If we use a `BTreeSet`, it will return the keys sorted, rather than in a random order.

Reply:
That is true, that does not really matter, but we can do it that way.